bin/drush-install.sh

```diff
@@ -54,3 +54,13 @@ if [ -n "$WITHSAMPLE" ]; then
   drush "$@" cvapi extension.install keys=org.civicrm.hrsampledata
   set +ex
 fi
+
+##################################
+## Set Default localisation settings
+LOC_FILE="en_US"
+if wget -q "https://download.civicrm.org/civicrm-l10n-core/mo/en_GB/civicrm.mo" > /dev/null; then
```

Comment:
I defined LOC_FILE="en_US" before the if so if it fail .. it use en_US which is the local default